pkgs/top-level/python-packages.nix

```
@@ -181,6 +181,8 @@ in {
 
   argon2_cffi = callPackage ../development/python-modules/argon2_cffi { };
 
+  archinfo = disabledIf (!isPy3k) (callPackage ../development/python-modules/archinfo { });
```

please move disable logic into expression
```suggestion
  archinfo = disabledIf (!isPy3k) (callPackage ../development/python-modules/archinfo { });
```